### PR TITLE
*: Cut release 1.9.0

### PR DESCRIPTION
**Pull request by LiliC**
After 7 days we had no bug reports, so time to cut the 1.9.0 release 🎉 

cc @brancz @tariq1890

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v1.9.0 / 2019-12-20
+
+After a testing period of 7 days, there were no additional bugs found or features introduced.
```
Comment (LiliC) [inline CHANGELOG.md]: Did we want to say something about this being the last release in the CHANGELOG or is this the wrong place for it?

Comment (tariq1890) [reply CHANGELOG.md]: I don't think we need it in the `CHANGELOG` IMO.

**Comment by LiliC** [comment]
@serathius do you mind uploading the `k8s.gcr.io/kube-state-metrics:v1.9.0` image, thanks!

**Comment by serathius** [comment]
Sure.

**Pull request merged**